mozjs/build.rs

```diff
@@ -82,9 +82,9 @@ fn main() {
 
     if msvc {
         builder = builder.clang_args([
+            "-fms-compatibility",
             &format!("-FI{}", confdefs_path.to_string_lossy()),
             "-DWIN32",
-            "-GR-",
```

Comment:
Is removing this still needed?

Reply:
I believe so. Either way, `-fms-compatibility` effectively includes that by default.

Reply:
I would keep it to make things more explicit.

Reply:
Since we're compiling for Windows with MSVC, I'd say just having `-fms-compatibility` is fine. If anything further breaks, that's a clang thing.

Moreover, this lines up with `mozjs-sys/build.rs`, which also uses this flag.

Reply:
I think (not sure) that those options were historically sourced from SM, and there is -GR- specified explicitly, that's why I would keep it. @jdm might know this more specifically.